### ISSUE TITLE
settings to reproduce table 3

### ISSUE THREAD

**Issue by peymanrostami** [irwinherrmann/stochastic-gates#4]
Dear @irwinherrmann,

Thank you for the code and for the description so far.

I would like to reproduce the result of table 3 (where t = .6 of the activations, using dependent gates should give 94.36% accuracy for ResNet110_cifar).

So far I could only obtain 88% accuracy (using ResNet110_cifar as you mentioned previously) while playing with and optimizing the training parameters myself. But since this is a large model, optimizing it this way will be time consuming.

May I ask you to kindly provide the **input arguments** or **pre-trained wights (if any)** to reproduce the result for table 3?

Thank you again,
Peyman

**Comment by irwinherrmann** [comment]
What accuracy are you getting for the t= 1 network?

If possible, you should try the same setup as the paper -- for cifar, use a clean network to start a fine-tuning run. This paper is quite old and unfortunately I don't have any of the CIFAR weights saved.

**Comment by peymanrostami** [comment]
Thank you for your reply. When setting t=1,  I am getting best accuracy = 91.2700% on cifar 10, using resnet 110. And, this is the result of training for 300 epochs starting with a lr=0.1 and then reducing the lr every 60 epochs with a factor of 0.1.

But the baseline should be some 93.21% according to table 1 of  [this paper](https://openaccess.thecvf.com/content_CVPR_2019/papers/Zhao_Variational_Convolutional_Neural_Network_Pruning_CVPR_2019_paper.pdf)  and 93.39% based on table 3 of your own paper.

So, some 2% accuracy is missing probably because I am using a different version of Pytorch?

Thank you again,
I will try to change the parameters a little and see if I can reproduce the result for the baseline, if I succeed, I will post the procedure here.
